program/find-sum-of-n-natural-numbers-using-formula/FindSumOfNNaturalNumbersUsingFormula.ts

```diff
@@ -0,0 +1,4 @@
+const FindSumOfNNaturalNumbersUsingFormula = (n: number): number => {
+    return (n * (n + 1)) / 2;
+}
+export default FindSumOfNNaturalNumbersUsingFormula;
```

Comment:
Looks good, but can you add `console.log` for the function to get the answer print in console?

Reply:
sure I have added the required console log

Reply:
Add `console.log` outside the function instead of inside. Like `console.log(FindSumOfNNaturalNumbersUsingFormula(5))` at line 6. 

Thank you!

Reply:
Made the required changes